### ISSUE TITLE
go get failing 

### ISSUE THREAD

**Issue by joeblew99** [myitcv/immutable#52]
playing around with your react stuff, but ...

````
x-MacBook-Pro:~ apple$ go get -u myitcv.io/immutable
# cd /Users/apple/workspace/go/src/myitcv.io/immutable; git submodule update --init --recursive
fatal: No url found for submodule path '_vendor/src/github.com/myitcv/gogenerate' in .gitmodules
package myitcv.io/immutable: exit status 128

x-MacBook-Pro:~ apple$ go get -u github.com/myitcv/immutable
# cd /Users/apple/workspace/go/src/github.com/myitcv/immutable; git submodule update --init --recursive
fatal: No url found for submodule path '_vendor/src/github.com/myitcv/gogenerate' in .gitmodules
package github.com/myitcv/immutable: exit status 128

````

**Comment by myitcv** [comment]
Apologies; a remnant of the setup pre move to `myitcv.io/*`

Fixed in #53

**Comment by joeblew99** [comment]
cheers thank Paul :)
